tests/functional/contribute-stories.js

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public" + '/n' +
+ "* License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* global casper */
+'use strict';
+
+var config = require('../lib/config');
+var helpers = require('../lib/helpers');
+var path = '/contribute/stories/';
+var url = config.base() + path;
+
+casper.test.begin('Contribute Stories, Elements: ' + url, 11, function suite(test) {
+    casper.start(url, function() {
+        test.assertHttpStatus(200);
+
+        test.assertVisible('#contribute-nav-menu .nav-cta', 'Get involved button is visible');
+        test.assertVisible('.head-stats', 'Community stats visible');
+
+        test.assertVisible('.story-intro .photo', 'The Mozillian profile image is visible');
+        test.assertVisible('.story-areas', 'Areas of activity content visible');
+        test.assertVisible('.story-links', 'List of links to social and other external links visible');
+
+        test.assertElementCount('.stories-other .person', 3, 'List of three Mozillians visible');
+        test.assertElementCount('.person a.url', 3, 'Each Mozillians read more link is visible');
+
+        test.assertVisible('.extra-event .event-link', 'There is one featured event');
+        test.assertVisible('.extra-event .events-all', 'All events link visible');
+        test.assertVisible('#newsletter-form', 'The newsletter form is visible.');
+    });
+
+    casper.run(function() {
+        test.done();
+        helpers.done();
+    });
+});
+
+casper.test.begin('Contribute Stories, Individual Story Page: ' + url, 5, function suite(test) {
+    casper.start(url, function() {
+        this.click('.stories-other .person:first-child a.url');
+    });
+
+    casper.waitForUrl(/contribute\/stories\/shreyas\/$/, function() {
```

Comment:
Better just to load this page directly maybe? This seems like a separate test going on here , rather than anything directly related to the content on `/contribute/stories/`. I don't think verifying that a link from one page to the other is working really adds much, seeing as we have tests for both pages individually?